javalib/src/main/scala/java/lang/String.scala

```diff
@@ -8,7 +8,8 @@ import java.util._
 import java.util.regex._
 import java.nio._
 import java.nio.charset._
-import java.util.Objects
+import java.util.{Objects, Arrays}
```

Comment:
Unused import?

```suggestion
import java.util.Objects
```

Reply:
@tanishiking 

Thank you for the review & suggestion.  Good to have a second pair of eyes.